proto/sentry_protos/sentry/v2/taskbroker.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
+
+package sentry_protos.sentry.v2;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+enum OnAttemptsExceeded {
+  ON_ATTEMPTS_EXCEEDED_UNSPECIFIED = 0;
+  ON_ATTEMPTS_EXCEEDED_DISCARD = 1;
+  ON_ATTEMPTS_EXCEEDED_DEADLETTER = 2;
+}
+
+message RetryState {
+  // Current attempt number
+  uint32 attempts = 1;
+
+  // The classname or adapter type for the retry policy
+  string kind = 2;
+
+  // After this number of attempts, the task is either discarded or deadlettered.
+  uint32 max_attempts = 3;
+
+  // The action to take after the attempts_limit is exceeded.
+  OnAttemptsExceeded on_attempts_exceeded = 4;
+
+  // Whether a task should be executed at most once.
+  optional bool at_most_once = 5;
+}
+
+enum TaskActivationStatus {
+  TASK_ACTIVATION_STATUS_UNSPECIFIED = 0;
+  TASK_ACTIVATION_STATUS_PENDING = 1;
+  TASK_ACTIVATION_STATUS_PROCESSING = 2;
+  TASK_ACTIVATION_STATUS_FAILURE = 3;
+  TASK_ACTIVATION_STATUS_RETRY = 4;
+  TASK_ACTIVATION_STATUS_COMPLETE = 5;
+}
+
+// Task message that is stored in Kafka.
+// Once consumed, TaskActivations are wrapped with InflightActivation to track
+// additional state
```

Comment:
I don't know if we need to mention `InflightActivation` here as that structure is only internal to taskbroker right now.